projects/plugins/jetpack/tests/e2e/lib/env/global-setup.js

```diff
@@ -4,7 +4,7 @@ import TunnelManager from './tunnel-manager';
 module.exports = async function () {
 	// Create tunnel. Make it global so we can access it in global-teardown
 	global.tunnelManager = new TunnelManager();
-	await global.tunnelManager.create( process.env.SKIP_CONNECT );
+	await global.tunnelManager.create();
```

Comment:
Yes, but keep in mind it's still work in progress.
Running with `SKIP_CONNECT=true` breaks the tunnel, meaning it will create a new tunnel with a new subdomain, but the tests will use the old subdomain from local file (if it exists). I think this is an effect of the previous PR, when I moved the tunnel creation from beforeAll to global setup, so it doesn't get created for each test. I guess the idea was to have a different subdomain for the connection tests, even though a subdomain already exists in a file to be reused? If true, this may not work like this now, although the tests were passing after the change. 
Anyway, I'm looking into how to accommodate all of these, but for the moment I'll revert this, it was only a local experiment.